java-tools/OsmAndServer/src/main/java/net/osmand/server/controllers/pub/UserdataController.java

```diff
@@ -241,10 +239,19 @@ public ResponseEntity<String> userRegister(@RequestParam(name = "email") String
 
 	@PostMapping(value = "/device-register")
 	public ResponseEntity<String> deviceRegister(@RequestParam(name = "email", required = true) String email,
-			@RequestParam(name = "token", required = true) String token,
-			@RequestParam(name = "deviceid", required = false) String deviceId) throws IOException {
+												 @RequestParam(name = "token", required = true) String token,
+												 @RequestParam(name = "deviceid", required = false) String deviceId,
+												 @RequestParam(name = "brand", required = false) String brand,
+												 @RequestParam(name = "model", required = false) String model,
+												 @RequestParam(name = "lang", required = false) String lang,
+												 @RequestHeader("User-Agent") String userAgent
+	) throws IOException {
 		String accessToken = UUID.randomUUID().toString();
-		return userdataService.registerNewDevice(email, token, deviceId, accessToken);
+		StringBuilder info = new StringBuilder(userAgent == null ? "OsmAnd" : userAgent);
```

Comment:
No need user agent (i.e. Version / OsmAnd+) in info

Reply:
done